### PR TITLE
docs: add FAQ page with support for dashboard not working

### PR DESCRIPTION
**Pull request by nicufk**
## Pull request description 
Related to https://github.com/kubeshop/testkube/issues/2112

## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

**Review by vsukhin — APPROVED** [review]

File: docs/docs/12-FAQ.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+### Why UI is not working or doesn't have results?
```
Comment (jfermi) [inline docs/docs/12-FAQ.md:1]:
```suggestion
### Why is the UI not working or does not return results?
```

File: docs/docs/12-FAQ.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- Make sure the api endpoint is configured
```
Comment (jfermi) [inline docs/docs/12-FAQ.md:2]:
```suggestion
- Make sure the API endpoint is configured.
```

File: docs/docs/12-FAQ.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+![img.gif](img/check-dashboard-api-endpoint.gif)
+
+- Make sure endpoint is providing data, e.g. accessing executors path
```
Comment (jfermi) [inline docs/docs/12-FAQ.md:6]:
```suggestion
- Make sure the endpoint is providing data, e.g. accessing the executors path.
```

File: docs/docs/12-FAQ.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+```sh
+curl <endpoint>/v1/executors 
+```
+
+- If it is not providing data make sure that all the testkube components are running properly
```
Comment (jfermi) [inline docs/docs/12-FAQ.md:12]:
```suggestion
- If no data is provided, make sure that all the Testkube components are running properly.
```

File: docs/docs/12-FAQ.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+kubectl get pods -n testkube
+NAME                                                        READY   STATUS    RESTARTS   AGE
+pod/testkube-api-server-8445fd7b9f-jq5rh                    1/1     Running   0          10d
+pod/testkube-dashboard-99f4c6cf5-x4dkz                      1/1     Running   0          12d
+pod/testkube-minio-testkube-76786f8f64-9nl4c                1/1     Running   1          24d
+pod/testkube-mongodb-74587998bb-8pzl2                       1/1     Running   0          12d
+pod/testkube-operator-controller-manager-77ffbb8fdc-rxhvx   2/2     Running   0          5d23h
+```
+
+- If any of this doesn't help contact us on [Discord](https://discord.com/invite/6zupCZFQbe)
```
Comment (jfermi) [inline docs/docs/12-FAQ.md:24]:
```suggestion
- If these options do not solve the problem, please contact us on [Discord](https://discord.com/invite/6zupCZFQbe).
```

**Review by jfermi — COMMENTED** [review]
Editing suggestions.

**Review by fivenp — APPROVED** [review]
LGTM

File: docs/docs/12-FAQ.md (hunk 1 above)
Comment (fivenp) [inline docs/docs/12-FAQ.md:1]:
```suggestion
# FAQ

Frequently asked questions regarding your Testkube installation.

### Why is the UI not working or does not return results?
```

**Pull request merged**